apps/angularlogos/src/app/app.component.html

```diff
@@ -1,15 +1,21 @@
 <div class="main">
   <div class="content">
     <div class="intro">
-      <h1><a routerLink="/">Angular Logos</a></h1>
+      <h1>
+        <a routerLink="/">
+          <img src="/assets/angular.svg" alt="Angular Logo" />
+          Angular Logos
+        </a>
+      </h1>
       <p>
-        Welcome to Angular Logos a website dedicated to try and collect all logos, badges that are inspired on the original Angular logo.<br />
-        The logos that can be found on this website can be related to a conference, MeetUp group, NPM package or anything else that exists
+        <strong>Welcome to Angular Logos</strong> – a website dedicated to collecting all logos and badges that are inspired by the original
```

Comment:
`dedicated to collection` --> `dedicated to collect`

Reply:
> “Dedicated to” is followed by the gerund; “helping.” “She’s dedicated to helping the poor.” This also applies to the noun form, dedication. For example, “I’m impressed by her dedication to helping the poor.”

https://www.espressoenglish.net/to-followed-by-ing/

Reply:
Still sounds so weird 😛

Reply:
I know what you mean 😉 asked a native speaker and she confirmed